app/controllers/users/noticed_notifications_controller.rb

```diff
@@ -11,7 +11,7 @@ def index
   def mark_as_read
     notification = NoticedNotification.find(params[:notification_id])
     notification.update(read_at: Time.zone.now)
-    project = Project.find(notification.params[:project_id])
+    project = notification.params[:project]
```

Comment:
What is the difference does this change make ?

Reply:
So previously we were finding the record using `project_id` in `Project` but now the object is already saved in `params[: project]` so there is no need to find it and complicate it once again!